sql/src/main/java/io/crate/jobs/PageBucketReceiver.java

```diff
@@ -56,6 +56,14 @@
  */
 public interface PageBucketReceiver {
 
+    /**
+     * Receives a bucket from an upstream which holds result data. This method may
+     * be called multiple times with the same bucketIdx if there are multiple pages.
```

Comment:
think this could be understand wrongly as calling this with the same bucketIdx on the same page will fail.

Reply:
As far as I understand, for every page it can be called once but wouldn't that be implementation detail also? :)

Reply:
yes so i'd rephrase it or leave it out